assignment/api/src/resources/assignment_fr_FR.properties

```diff
@@ -250,7 +250,7 @@ gen.theassall6=This assignment allows submissions using the text box below only.
 gen.yoursubwill=Your submission will be sent to {0} to be reviewed for plagiarism.
 gen.yoursubwill.indexed=Your submission will be sent to {0} to be reviewed for plagiarism. It will be indexed to be compared against by other submissions.
 gen.subStudentPreview=This assignment has enabled student preview which allows students to submit as many papers as a draft as they wish. A draft paper report is only visible to the owner of the paper, however instructors will be able to see the number of draft submissions and the report scores of each draft. Also, if there is a due date, the most recent draft will automatically be submitted if the student has no other submissions. Students can submit a draft paper as their final submission at any time within the plagiarism report.
-gen.onlythefoll=Only the following file types will be accepted\: {0}
+gen.onlythefoll=<li>Only the following file types will be accepted\: {0}.</li>
```

Comment:
All of these English messages should probably be removed from this French file.

Reply:
The entire French file is in english, I think it's better to have an English message than a not found string error?

Reply:
If the French file contains no French, it should just be deleted. If a string cannot be found for a given locale, it automatically falls back to the default locale anyway (en_US). So duplicating them in multiple locale files is redundant and unnecessary.

Reply:
Okay, I'll delete the string.